qiskit_nature/circuit/library/ansaetze/uccsd.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+The UCCSD variational form.
+"""
+
+from typing import Optional, Tuple
+
+import logging
+
+from qiskit_nature.operators.second_quantization.qubit_converter import QubitConverter
+from .ucc import UCC
+
+logger = logging.getLogger(__name__)
+
+
+class UCCSD(UCC):
+    """The UCCSD Ansatz.
+
+    This is a convenience subclass of the UCC Ansatz.
+    """
+
+    def __init__(self, qubit_converter: Optional[QubitConverter] = None,
+                 num_particles: Optional[Tuple[int, int]] = None,
+                 num_spin_orbitals: Optional[int] = None,
+                 reps: int = 1):
+        """
+
+        Args:
+            qubit_converter: the QubitConverter instance which takes care of mapping a
+            :code:`~.SecondQuantizedOp` to a :code:`~.PauliSumOp` as well as performing all
```

Comment:
What's the difference of `:code:` to using 
```
`
```
?

Reply:
In cases like the above, since its referring to a class we always used `:class:`. `:meth:` when referring to a method etc.